react-native-ssi/ios/RNSsi.mm

```diff
@@ -42,6 +42,53 @@ @implementation RNSsi
     }
 }
 
+RCT_REMAP_METHOD(signVerifiableCredentialJWT,
+                 keyId:(NSString *)keyId
+                 privateJwk:(NSDictionary *)privateJwk
+                 verifiableCredential:(NSDictionary *)vc
+                 resolver:(RCTPromiseResolveBlock)resolve
+                 rejecter:(RCTPromiseRejectBlock)reject)
+{
+    NSError *error;
+    NSData *vcBytes = [NSJSONSerialization dataWithJSONObject:vc options:NSJSONWritingPrettyPrinted error:&error];
+    if (error != nil) {
+        return reject(@"RNSsi", @"error serializing vc bytes", error);
+    }
+
+    NSData *jwkBytes = [NSJSONSerialization dataWithJSONObject:privateJwk options:NSJSONWritingPrettyPrinted error:&error];
+    if (error != nil) {
+        return reject(@"RNSsi", @"error serializing jwk bytes", error);
+    }
+
+    NSString *result = SsiSignVerifiableCredentialJWT(keyId, jwkBytes, vcBytes, &error);
```

Comment:
I see you went with a string after all. any particular reason?

Reply:
The sign function had always returned a string for the JWT. I just left as-is for now.